independent-publisher-connectors/My Acclaro/apiProperties.json

```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
```

Comment:
@msanz-acclaro But the connection is configured as "API Key" in the swagger, Did you test the connector with the latest files ?

Reply:
My bad, forgot to update this file